### PR TITLE
Spectron integration tests config

### PR DESCRIPTION
**Pull request by JacekWalczak92**

### CODE REVIEW

File: src/components/BalanceBanner/index.tsx [hunk 1]
```diff
@@ -131,9 +131,9 @@ function BalanceBanner(props: Props) {
                     ) : null}
                 </AddressInfo>
                 {delegatedAddress && (
-                    <DelegateContainer>
+                    <DelegateContainer id="delegate">
                         <Delegate>{t('components.balanceBanner.delegated_to')}</Delegate>
-                        <TezosAddress address={delegatedAddress} color="white" size={ms(-1)} weight={300} />
+                        <TezosAddress id="tezosAddres" address={delegatedAddress} color="white" size={ms(-1)} weight={300} />
```
Comment (anonymoussprocket) [inline src/components/BalanceBanner/index.tsx]: spelling

File: src/containers/LoginCreate/index.tsx [hunk 2]
```diff
@@ -57,8 +57,18 @@ function LoginCreate() {
         if (event.detail === 0 && walletLocation && walletFileName) {
             return;
         }
+
+        // [TESTING]
+        if (ipcRenderer.sendSync('is-spectron')) {
```
Comment (anonymoussprocket) [inline src/containers/LoginCreate/index.tsx:62]: Does this code need to live in the release?

File: tsconfig.json [hunk 3]
```diff
@@ -26,7 +26,7 @@
   },
   "include": [
     "src",
-    "./test"
+    "./test", "test"
```
Comment (anonymoussprocket) [inline tsconfig.json:29]: is this duplicated?

File: test/spec.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
+const assert = require('assert');
```
Comment (anonymoussprocket) [inline test/spec.js]: tests should be organized in a directory structure similar to the code. Consider how ConseilJS does it: [code](https://github.com/Cryptonomic/ConseilJS/tree/master/src) vs [tests](https://github.com/Cryptonomic/ConseilJS/tree/master/test).

**Review by anonymoussprocket — CHANGES_REQUESTED** [review]

**Comment by vishakh** [comment]
Closing in favor of #346.

**Pull request closed without merging**